### ISSUE TITLE
flutter build apk issue

### ISSUE THREAD

**Issue by smartdev008** [Tencent-RTC/TUIRoomKit#562]
I got error like this when flutter buid apk --release in conference example project
my flutter sdk version is 3.22.3

Dart snapshot generator failed with exit code -1073740940

Target android_aot_release_android-arm failed: Exception: AOT snapshotter exited with code -1073740940

![image](https://github.com/user-attachments/assets/4270bd10-3e1f-4170-93a4-ec10fe5b8a68)
![Screenshot_2](https://github.com/user-attachments/assets/10ae598b-793f-46a9-98b8-f6ef5aafa36a)

**Comment by PualGeorge** [comment]
Does your problem still exist after removing .gradle file and then again flutter clean and flutter pub get?

**Comment by prituljogiya** [comment]
no, I have not been facing an issue in some time, is there a due grade update .. rather than you can you fvm
